src/components/Logout.jsx

```diff
@@ -0,0 +1,23 @@
+import { useNavigate } from "react-router-dom";
+
+
+export const Logout = () => {
+
+    const nav = useNavigate();  
+    const handleLogout = () =>{
+
+        sessionStorage.removeItem('token');
+        sessionStorage.removeItem('role'); 
+        nav('/'); 
+    }  
+
+  return (
+    <button
+        data-testid="logout" 
```

Comment:
_logout_ is too generic for a `data-testid`. You should quickly tell which element at the UI level corresponds to a particular id.

Reply:
_logout-button_ can be an option :)

Reply:
I agree with you